### ISSUE TITLE
article selection broken on FireFox

### ISSUE THREAD

**Issue by filmenczer** [IUNetSci/hoaxy-botometer#262]
See https://github.com/IUNetSci/hoaxy-frontend/issues/1

**Comment by benabus** [comment]
I don't believe this is a Firefox error.  I was unable to replicate the error in Firefox.

Looking at the errors that show up in the console in the video attached to the original issue, it looks like the twitter share button library wasn't loaded properly.  This could be for a number of reasons.  It might have something to do, potentially with Adblock.  There were also notifications regarding poor performance (undoubtedly related to the large graph).  

Regardless of why this error occurred for this user, I believe the underlying problem is that the twitter library wasn't loaded properly and this should be mitigated by doing a simple check for twttr being undefined.

**Comment by filmenczer** [comment]
Okay, closing.

**Comment by filmenczer** [comment]
Thank you, sorry for my misunderstanding.
